sources/packages/backend/libs/services/src/disbursement-schedule/disbursement-schedule-shared.service.ts

```diff
@@ -129,7 +129,7 @@ export class DisbursementScheduleSharedService extends RecordDataModelService<Di
       assessment.modifier = auditUser;
       // Adjust the saved grants disbursements with the values already disbursed.
       // !Intended to process only grants (CanadaGrant/BCGrant)
-      this.applyGrantsAlreadyDisbursedValues(
+      await this.applyGrantsAlreadyDisbursedValues(
```

Comment:
I believe that the method is wrongly marked with `async`. Please check if the `async` can be removed from the method and remove the `await` add here.

Reply:
There are multiple promises being resolved inside the anonymous function passed as an argument to `this.dataSource.transaction`, therefore I feel `async` is required in this case.